internal/config/config.go

```diff
@@ -215,11 +215,11 @@ func Get() *IngressConfig {
 		IngressCfg.KafkaConfig.KafkaSSLConfig.KafkaUsername = options.GetString("KafkaUsername")
 		IngressCfg.KafkaConfig.KafkaSSLConfig.KafkaPassword = options.GetString("KafkaPassword")
 		IngressCfg.KafkaConfig.KafkaSSLConfig.SASLMechanism = options.GetString("SASLMechanism")
-		IngressCfg.KafkaConfig.KafkaSSLConfig.Protocol = options.GetString("Protocol")
 	}
 
 	if options.IsSet("KafkaCA") {
 		IngressCfg.KafkaConfig.KafkaSSLConfig.KafkaCA = options.GetString("KafkaCA")
+		IngressCfg.KafkaConfig.KafkaSSLConfig.Protocol = options.GetString("Protocol")
```

Comment:
I think this needs to be moved outside of the if statement.  If we set the default setting to "plaintext", then I dont think we'll need to have a `options.IsSet("Protocol")` check...we could just set the `IngressCfg.KafkaConfig.KafkaSSLConfig.Protocol`  setting to whatever in in the options.  If we do not default this to "plaintext", then we should probably wrap setting the `IngressCfg.KafkaConfig.KafkaSSLConfig.Protocol` in an `if options.IsSet("Protocol")` check.